src/integration/kotlin/com/github/imflog/schema/registry/tasks/download/DownloadTaskIT.kt

```diff
@@ -93,6 +100,69 @@ class DownloadTaskIT : KafkaTestContainersUtils() {
         Assertions.assertThat(resultFile2.readText()).contains("description")
     }
 
+    @ParameterizedTest(name = "{0}")
+    @ArgumentsSource(SchemaArgumentProvider::class)
+    fun `Should format supported schema types when pretty is specified`(type: String, oldSchema: ParsedSchema,
+                                                                        newSchema: ParsedSchema) {
+        // Given
+        val subjectName = "parameterized-$type"
+
+        client.register(subjectName, oldSchema)
```

Comment:
To harden the test, can you remove all the new line when registering the schema ?
I don't know if the mock registry client do any formatting on the passed schemas.

Reply:
Good idea, will do.